Update See Also section of man page to remove man pages not found on Solaris

--- luit-20230201/luit.man
+++ luit-20230201/luit.man
@@ -433,19 +433,12 @@ Selecting alternate sets of control char
 will never be.
 .\" ***************************************************************************
 .SH SEE ALSO
-These are portable:
 .bP
 xterm(__mansuffix__),
 .bP
-ncurses(3X).
-.PP
-These are Linux-specific:
-.bP
-unicode(__miscmansuffix__),
-.bP
-utf-8(__miscmansuffix__),
+ncurses(3X),
 .bP
-charsets(__miscmansuffix__).
+locale(__miscmansuffix__).
 .PP
 These are particularly useful:
 .nf
